src/commands/dev-env-sync-sql.js

```diff
@@ -174,14 +176,19 @@ export class DevEnvSyncSQLCommand {
 			await unzipFile( this.gzFile, this.sqlFile );
 			console.log( `${ chalk.green( '✓' ) } Extracted to ${ this.sqlFile }` );
 		} catch ( err ) {
-			await this.track( 'archive_extraction_error', { errorMessage: err.message } );
+			await this.track( 'error', { error_type: 'archive_extraction', errorMessage: err?.message, stack: err?.stack } );
 			exit.withError( `Error extracting the SQL export: ${ err.message }` );
 		}
 
 		try {
 			console.log( 'Extracting site urls from the SQL file...' );
 			this.siteUrls = await extractSiteUrls( this.sqlFile );
 		} catch ( err ) {
+			await this.track( 'error', {
+				error_type: 'extract_site_urls',
+				errorMessage: err?.message,
```

Comment:
```suggestion
				error_message: err?.message,
```

Tracks will reject this (although Pendo will accept it)  because it was in camel case . Tracks only accepts snake_case

Reply:
This will need to be changed everywhere in this file

Reply:
TIL